core/storage/feedback/gae_models_test.py

```diff
@@ -313,6 +351,63 @@ def test_export_data_nontrivial(self) -> None:
 
         self.assertEqual(test_data, user_data)
 
+    def test_get_messages(self) -> None:
```

Comment:
Please give this a better name, I don't know what *behaviour* this is testing exactly.

Reply:
Updated it now